src/Compilers/CSharp/Portable/Symbols/MemberSignatureComparer.cs

```diff
@@ -602,7 +602,7 @@ private static bool HaveSameParameterTypes(ImmutableArray<ParameterSymbol> param
                 {
                     if (!type1.Equals(type2, ignoreDynamic: true) ||
                         !param1.CustomModifiers.SequenceEqual(param2.CustomModifiers) ||
-                        (!param1.CustomModifiers.IsEmpty && param1.RefKind != RefKind.None && param2.RefKind != RefKind.None && param1.HasByRefBeforeCustomModifiers != param2.HasByRefBeforeCustomModifiers))
+                        (param1.RefKind != RefKind.None && param2.RefKind != RefKind.None && param1.CountOfCustomModifiersPrecedingByRef != param2.CountOfCustomModifiersPrecedingByRef))
```

Comment:
Could you omit the test `param1.RefKind != RefKind.None && param2.RefKind != RefKind.None &&` here?